airbyte-webapp/src/packages/cloud/services/thirdParty/launchdarkly/LDExperimentService.tsx

```diff
@@ -123,7 +99,12 @@ const LDInitializationWrapper: React.FC<React.PropsWithChildren<{ apiKey: string
   };
 
   if (!ldClient.current) {
-    ldClient.current = LDClient.initialize(apiKey, mapUserToLDUser(user, locale, null));
+    const userContext = createUserContext(user, locale);
+    const contexts = workspaceId
+      ? createMultiContext(userContext, createWorkspaceContext(workspaceId))
+      : createMultiContext(userContext);
```

Comment:
Minor code style nitpick (take or leave): Since it's the very same code as L148ff it might make sense to just create a `createContext(user, locale, workspaceId?)` that returns the final multi context for us, so we can make sure it's never diverging from each other.